### DISCUSSION TITLE
Can be set heavier/thicker base fonts?

### DISCUSSION THREAD

**Discussion by luc0x61** [sumatrapdfreader/sumatrapdf#4190] [Q&A]
Can I set base fonts to be heavier/thicker in weight?
I'm just comparing some PDFs, opened with Adobe Reader, with which the base text is heavier. I find that approach to show text more like a bold printed book, easier to read for *my* eyes (and my bad sight, of course).

**Comment by kjk** [comment]
Font rendering in PDFs is done by mupdf and I don't have the expertise to make such changes.
